CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+- Updated to LiveView 0.18.18
```

Comment:
This definately needs a minor version bump instead of a patch level and we should mention that it drops support of older version, right?

Reply:
We should, yes. I was actually too lazy to find out exactly which version of `phoenix_live_view` would still be enough to run this (e.g. when `to_form` was introduced). LiveView is changing too quickly..